src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenSource.cs

```diff
@@ -22,6 +22,7 @@ namespace System.Threading
     /// concurrently from multiple threads.
     /// </para>
     /// </remarks>
+    [DebuggerDisplay("IsCancellationRequested = {IsCancellationRequested}; Disposed = {_disposed}")]
```

Comment:
Is there prior art somewhere in runtime where we show disposed in a debugger display?

We also don't include it in CancellationToken's, and there's not really a difference here with regards to whether you're looking at the CTS directly or via a CT that's just wrapping it (a CT itself has no state other than the CTS with which it's associated).

I think this should just be IsCancellationRequested.

Reply:
@JamesNK Thoughts, as you asked for disposal in the original issue?

Reply:
FWIW I can't see any prior art for that in a quick GitHub search of runtime or aspnetcore.

Reply:
The reason I like having Disposed in the debug output for CTS is I've seen many bugs caused by a type disposing the CTS, and then getting an `ObjectDisposedException` when accessing `_cts.Token`.

I'm ok with Dispose being removed from the debug string if it isn't typical.

@stephentoub Has the runtime team ever thought about making `CancellationTokenSource.Token` accessible even after dispose?

Reply:
@stephentoub Has the runtime team ever thought about making `CancellationTokenSource.Token` accessible even after dispose? Quiet often I end up doing this:

```cs
private readonly CancellationTokenSource _cts = new();
private readonly CancellationToken _cancellationToken = _cts.Token;
```

That then allows code to access the token without worrying about ODE. It would be less code if `_cts.Token` could be used directly and be less surprising to folks who are new to use CTS.

Reply:
> Has the runtime team ever thought about making CancellationTokenSource.Token

To my recollection it hasn't come up. We could consider it, though.

Reply:
Issue for your consideration: https://github.com/dotnet/runtime/issues/105811